src/app/side-bar/dashboard/dashboard.component.html

```diff
@@ -9,7 +9,7 @@
   </div>
   <p fxLayoutAlign="center center">Days [2022]</p>
   <div fxLayout="column">
-    <div fxlayout="row" fxLayoutAlign="space-between center" class="dashboard-container__card-title">
+    <div fxFlex="20px"l fxlayout="row" fxLayoutAlign="space-between center" class="dashboard-container__card-title p-0 p-20">
```

Comment:
we usually use flex value as a percentage...please avoid using unit..... and please remove **fxFlex="20px"l** that extra l

Reply:
Resolved..............Thank you